app/views/experiments/_experiment_result.html.haml

```diff
@@ -1,8 +1,8 @@
 %h3.subheader= "Experiment result"
 
 - unless @experiment.results.nil?
```

Comment:
Jednak nie, kiedy jest pusta zawartość to wszystko wygląda ok, a sama sekcja "Experiment results" i tak się pojawiała zawsze przy supervised experiment.